cli/command/container/pause.go

```diff
@@ -34,8 +34,11 @@ func runPause(dockerCli *command.DockerCli, opts *pauseOptions) error {
 	ctx := context.Background()
 
 	var errs []string
+	errChan := parallelOperation(ctx, opts.containers, func(ctx context.Context, id string) error {
+		return dockerCli.Client().ContainerPause(ctx, id)
+	})
```

Comment:
Minor nit: I think you can just pass `dockerCli.Client().ContainerPause` directly as the last argument, since it already has the right signature